subsys/zigbee/cli/zigbee_cli_ctx_mgr.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <shell/shell.h>
+
+#include "zigbee_cli_ctx_mgr.h"
+
+
+/* The table to store context manager entries. */
+static struct ctx_entry zb_cli_ctx[CONFIG_ZIGBEE_SHELL_CTX_MGR_ENTRIES_NBR];
+
+/**@brief Delete context manager entry to mark it free and prepare to be
```

Comment:
These functions are already documented in the header, no need to duplicate it here.